zio-http/src/main/scala/zio/http/QueryParams.scala

```diff
@@ -18,9 +18,10 @@ package zio.http
 
 import java.nio.charset.Charset
 
-import zio.Chunk
+import zio.{Chunk, NonEmptyChunk}
```

Comment:
`NonEmptyChunk` is unused or?

Reply:
You are right - I left some unused imports, I would fix it